hedera-node/src/test/java/com/hedera/services/files/EntityExpiryMapFactoryTest.java

```diff
@@ -125,8 +125,9 @@ void toLongPropagatesNull() {
 
 	@Test
 	void throwsIaeOnNonsense() {
+		final var bytes = "wtf".getBytes();
 		// expect:
```

Comment:
nit: remove also comment (replace by a blank line)

Reply:
Many tests have comments for readability. If these are decided to be removed, might be better to do everything in separate PR.